src/lib/util/Config.ts

```diff
@@ -79,6 +90,31 @@ export class Config {
 		return this.configContent.engines.find(e => e.name === name);
 	}
 
+	public getOverriddenConfigPath(name: string): string {
```

Comment:
Adding and handling defaultConfig and overriddenConfigPath defined in Config.json.